internal/util/vault.go

```diff
@@ -184,21 +199,13 @@ func (kms *VaultKMS) GetPassphrase(key string) (string, error) {
 
 // SavePassphrase saves new passphrase in Vault.
 func (kms *VaultKMS) SavePassphrase(key, value string) error {
-	data, err := json.Marshal(map[string]map[string]string{
-		"data": {
+	data := map[string]interface{}{
+		"data": map[string]string{
 			"passphrase": value,
```

Comment:
TODO as an enhancement define `passphrase` as a string variable and use it in all functions?

Reply:
It is only used 2x, the same as `"data"` (for get/save). I do not think it makes things easier or clearer when there is a variable for these.